src/ui_manager.h

```diff
@@ -210,8 +211,9 @@ class ui_adaptor
 
         /* See the `ui_manager` namespace */
         static void invalidate( const rectangle<point> &rect, bool reenable_uis_below );
-        static void redraw();
-        static void redraw_invalidated();
+        static bool has_imgui();
+        static void redraw_all();
```

Comment:
I mean, fair, we can come up with a better name for these than just adding 'all', I just disagree calling it just 'redraw' since it doesn't specify what is being redrawn. maybe 'redraw_topmost_and_invalidated'?

Reply:
I don't have a better suggestion at the moment, but I think `redraw_all` is as misleading as `redraw`, and `redraw_topmost_and_invalidated` is a bit too long for my taste. And tbh not having a good name is why I haven't changed its name already since I implmented it.

Reply:
I deleted the new methods in ui_adaptor and renamed everything back to what it was, this should be fixed.